tests/test_spider.py

```diff
@@ -2,11 +2,38 @@
 import json
 from datetime import datetime
 import scrapy
-from scrapy.http.response.text import TextResponse
+from scrapy.http.response.html import HtmlResponse
 import textwrap
 from freezegun import freeze_time
 from urlobject import URLObject
 from pa11ycrawler.spiders.edx import EdxSpider
+try:
+    from urllib.parse import parse_qs
+except ImportError:
+    from urlparse import parse_qs
+
+
+CSRF_HEADER = (
+    "csrftoken=2JH7ojWIMGDjWxSrdnp4Jkg0bGxaS3MV; "
```

Comment:
NBD but I wonder if you want to make the token obviously-fake to avoid any confusion for readers.